pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -899,31 +965,36 @@ func validateSettings(ctx context.Context, needsRangeFeed bool, execCfg *sql.Exe
 	return nil
 }
 
-func getTableDescriptors(
+func getTargetDescriptors(
 	ctx context.Context,
 	p sql.PlanHookState,
 	targets *tree.BackupTargetList,
 	statementTime hlc.Timestamp,
 	initialHighWater hlc.Timestamp,
-) (map[tree.TablePattern]catalog.Descriptor, error) {
+) (
+	map[tree.TablePattern]catalog.Descriptor,
+	[]catalog.DatabaseDescriptor,
+	[]catalog.Descriptor,
```

Comment:
can you add names for these pls &/ change the return values to make them make more sense